frontend/integration-tests/protractor.conf.ts

```diff
@@ -96,7 +96,7 @@ export const config: Config = {
     olmUpgrade: ['tests/base.scenario.ts', 'tests/olm/update-channel-approval.scenario.ts'],
     performance: ['tests/base.scenario.ts', 'tests/performance.scenario.ts'],
     serviceCatalog: ['tests/base.scenario.ts', 'tests/service-catalog/service-catalog.scenario.ts', 'tests/service-catalog/service-broker.scenario.ts', 'tests/service-catalog/service-class.scenario.ts', 'tests/service-catalog/service-binding.scenario.ts'],
-    catalog: ['tests/base.scenario.ts', 'tests/catalog.scenario.ts'],
+    developerCatalog: ['tests/base.scenario.ts', 'tests/developer-catalog.scenario.ts'],
```

Comment:
I'd suggest just making this part of the `serviceCatalog` suite, particularly if it requires service catalog enabled to pass.